### PR TITLE
[UCSC-167] FE Post Single templates

### PR DESCRIPTION
**Pull request by bjcooper**
## What does this do/fix?

All the things for Post Single templates:

- Renames `Default_Image_Header_Template` to `Post_Single`.
- Builds out the Post Header block.
- Builds out the Press Inquiries block.
- Builds out Related Stories block.
- Block icons and normalized naming for all.
- Modifies the `core/post-terms` and `outermost/social-sharing` block output to add headings.
- Adds custom styles for those two blocks, and webpack config for building them.

### Links to relevant issues

- https://moderntribe.atlassian.net/browse/UCSC-167

### CODE REVIEW

File: src/Components/Press_Inquiries_Controller.php [hunk 1]
```diff
@@ -7,19 +7,36 @@
 class Press_Inquiries_Controller {
 
 	protected array $block;
+	protected string $panel_id;
 
 	public function __construct( $block ) {
-		$this->block = (array) $block;
+		$this->block    = (array) $block;
+		$this->panel_id = wp_unique_id( 'press-inquiries-block-' );
```
Comment (MlKilderkin) [inline src/Components/Press_Inquiries_Controller.php:14]: if I am not mistaken $block has unique ID already

File: src/views/patterns/social-sharing.php [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<?php declare(strict_types=1);
```
Comment (MlKilderkin) [inline src/views/patterns/social-sharing.php:1]: This won't work without BE proper registration

Comment (MlKilderkin) [reply src/views/patterns/social-sharing.php:1]: Do we actually need this?

**Review by MlKilderkin — APPROVED** [review]

File: src/views/patterns/social-sharing.php (hunk 2 above)
Comment (bjcooper) [reply src/views/patterns/social-sharing.php]: Nope, that's old. Good catch.

File: src/Components/Press_Inquiries_Controller.php (hunk 1 above)
Comment (bjcooper) [reply src/Components/Press_Inquiries_Controller.php]: Ah, yeah. I'll switch to just using the block ID.

File: src/views/related_stories_block/style.scss [hunk 3]
```diff
@@ -1,3 +1,83 @@
 /**
  * Related Stories Block Styles
  */
+
+@import "../../../assets/scss/mixins";
```
Comment (Vinsanity) [inline src/views/related_stories_block/style.scss:5]: just confirming that this doesn't need to change to `use` to avoid that issue from the other day.

**Review by Vinsanity — APPROVED** [review]
🟢 Great work here @bjcooper ! 🌮 just one comment to review but no blockers.

File: src/views/related_stories_block/style.scss (hunk 3 above)
Comment (bjcooper) [reply src/views/related_stories_block/style.scss:5]: Correct! The issue we had the other day was that the plugin was trying to @import mixins from the theme, but the theme is not present at the expected path during the deploy build process.

**Pull request merged**